config/config_test.go

```diff
@@ -135,6 +139,11 @@ func TestLoadConfigFileWithGroups(t *testing.T) {
 	assert.Equal(t, expected, conf)
 }
 
+func TestLoadInvalidConfigFile(t *testing.T) {
+	_, err := loadForFileFromBytes(sampleInvalidConfig, "foobar2000", nil)
+	assert.Equal(t, err.Error(), "error loading config: no creation rules found")
```

Comment:
I think it'd be best to just check that this returns an error, and not check the error message. Checking for error messages is pretty brittle and leads to breaking tests in my experience.